packages/jmix-react-ui/src/ui/table/DataTable.tsx

```diff
@@ -152,9 +152,8 @@ export interface ColumnDefinition<TEntity> {
    */
   columnProps: ColumnProps<TEntity>
 }
-class DataTableComponent<
-  TEntity extends object = object
-> extends React.Component<DataTableProps<TEntity>, any> {
+
+class DataTableComponent<TEntity extends object = object> extends React.Component<DataTableProps<TEntity>, any> {
```

Comment:
Please turn off automatic reformatting, too much changes unrelated to the ticket.